src/WebApiContrib.Formatting.MsgPack/packages.config

```diff
@@ -1,7 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.AspNet.WebApi.Client" version="4.0.20710.0" targetFramework="net40-Client" />
-  <package id="Microsoft.Net.Http" version="2.0.20710.0" targetFramework="net40-Client" />
-  <package id="MsgPack" version="0.1.0.2011042300" targetFramework="net40-Client" />

```

Comment:
I replaced MsgPack with MsgPack.Cli. The MsgPack package has not been updated since October of 2011 while MsgPack.Cli is under active development.